src/components/Developers/Developers.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import {Row,Container,Col} from 'react-materialize'
+import './Developers.css'
+
+class Developers extends Component {
+	render() {
+		//Const or fetch API
+		const users=[
```

Comment:
I can send the exact developer APIs JSON. So that maybe you can write your code accordingly if you want. Or maybe you can change later on.